app/views/course/assessment/answer/programming/_test_cases.html.slim

```diff
@@ -1,5 +1,5 @@
 - auto_grading = answer.auto_grading ? answer.auto_grading.actable : nil
-- is_grader = can?(:grade, answer.answer)
+- is_grader = can?(:read_tests, answer.answer)
```

Comment:
Ok. I had originally wanted to keep the scope as tight as possible. Will convert to a submission level permission.